ext/openssl/lib/openssl/ssl.rb

```diff
@@ -21,7 +21,7 @@ module OpenSSL
   module SSL
     class SSLContext
       DEFAULT_PARAMS = {
-        :ssl_version => "SSLv23",
+        :ssl_version => "TLSv1_2",
```

Comment:
I believe @jvehent is correct.  It's likely to still allow TLSv1.1 in the proposed config to be effective as "TLS 1.2 only".  I believe his proposal of the relative Ruby OpenSSL SSL_OP_NO_TLSv1_1 equivalent will achieve this, but the original goal of this was to rid ourselves of TLSv1.0.  I think taking the next step and deprecating TLSv1.1 should be something to consider.